types/factory-girl/index.d.ts

```diff
@@ -39,23 +48,32 @@ declare namespace factory {
         /**
          * Generates and returns model attributes as an object hash instead of the model instance
          */
-        attrs<T>(name: string, attrs?: Attributes<Partial<T>>): Promise<T>;
+        attrs<T>(name: string, attrs?: Attributes<Partial<T>>, buildOptions?: BuildOptions): Promise<T>;
 
         /**
          * Generates and returns a collection of model attributes as an object hash instead of the model instance
          */
-        attrsMany<T>(name: string, num: number, attrs?: ReadonlyArray<Attributes<Partial<T>>>): Promise<T[]>;
+        attrsMany<T>(
+            name: string,
+            num: number,
+            attrs?: MaybeReadonlyArray<Attributes<Partial<T>>>,
+            buildOptions?: BuildOptions | ReadonlyArray<BuildOptions>,
+        ): Promise<T[]>;
 
         /**
          * Builds a new model instance that is not persisted
          */
-        build<T>(name: string, attrs?: Attributes<Partial<T>>): Promise<T>;
+        build<T>(name: string, attrs?: Attributes<Partial<T>>, buildOptions?: BuildOptions): Promise<T>;
 
         /**
          * Builds an array of model instances that are persisted
          */
-        buildMany<T>(name: string, num: number, attrs?: Attributes<Partial<T>>): Promise<T[]>;
-        buildMany<T>(name: string, attrs?: ReadonlyArray<Attributes<Partial<T>>>): Promise<T[]>;
```

Comment:
As I investigated the implementation, `Factory#buildMany(...)` needs `count` parameter.

- https://github.com/simonexmachina/factory-girl/blob/f7ec9d5004021ff1334ab9440e9b5cc4fd1a18b0/src/Factory.js#L99